test/CVE.py

```diff
@@ -0,0 +1,22 @@
+import sys
+import subprocess
+
+def main():
+    cmd = sys.argv[2:]
+    proc = subprocess.Popen(cmd,
+                        stdout=subprocess.PIPE,
+                        stderr=subprocess.STDOUT)
+    stdout, stderr = proc.communicate()
+    print stdout
+    #print stderr
+    # We expect 1 error code is OK and we expect 134 or 
+    # similar exit code for a vulnerable failure
+    if proc.returncode == 1 or proc.returncode == 0:
+        print('zlib is not vulnerable to {0} ({1})'.format(sys.argv[1], proc.returncode))
+        exit(0)
+    else:
+        print('zlib VULNERABLE to {0} ({1})'.format(sys.argv[1], proc.returncode))
```

Comment:
"zlib is VULNERABLE to"

Reply:
Actually this is the way that the testCVEinputs.sh has it.

Reply:
As a teacher, I usually prefer proper English, but I don't always have time to go through all pull requests... Consistency is always good as long as it doesn't result in build or run-time errors :)

Reply:
Haha, no wonder I feel like I'm being graded on an exam! 😂 Yeah, I think here I will just keep it consistent, and if we want to later I can do another pull request for both files that corrects the grammer.